ASM/README.md

```diff
@@ -58,14 +58,16 @@ You'll need to disable `Unique Game Save Directory` in Project64 for these to wo
 
 How to use the Debug mode: 
 - First put the DEBUG_MODE variable at 1 in debug.h.
-- You will then have access to a hidden menu with the following options:
+- Now the N64 logo sequence at the start of the game will be skipped, the L button will allow you to levitaten and you will then have access to a hidden menu with the following options:
   - Instant warps to Dungeons, Bosses or Overworld locations
   - Item inventory edits
   - Instant age switch with the current location kept
-  - Bunny Hood applied on
-The menu will appear if you press either L+R or Dpad-Up according to what you set for the menu_not_on_dup variable in debug.c.
+  - Bunny Hood
+  - In-game clock
+  - Actor and overlay list
+  - Scene flags setters
+The menu will appear if you press either Dpad-Up or L.
```

Comment:
So this seems to have broken the ability to levitate? When I press L, Link will briefly jump up, but then be unable to continue levitating because the menu overrides the input ig. Why not keep the menu to L+R?

Reply:
Yeah i don't know why i tried to change it to be easier lol.
Went back to L+R, and fixed the levitation.